provider/provider.go

```diff
@@ -74,6 +74,7 @@ func New() *schema.Provider {
 			"coder_parameter":      parameterDataSource(),
 			"coder_git_auth":       gitAuthDataSource(),
 			"coder_external_auth":  externalAuthDataSource(),
+			"coder_user":           userDataSource(),
```

Comment:
This will only ever be populated with information regarding the workspace owner.
Should I rename it to `coder_workspace_owner` instead?